Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN apt-get update && apt-get install -y \
   build-essential \
   firefox-esr \
   libpoppler-cpp-dev \
+  nodejs \
```

Comment:
Noting John's objection to including nodejs and npm in the Dockerfile. It's fair.